docs/cog_guides/trivia.rst

```diff
@@ -0,0 +1,414 @@
+
+    [p]triviaset custom list
+
+**Description**
+
+List all uploaded custom trivia lists.
+
+.. _trivia-command-triviaset-custom-delete:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+triviaset custom delete
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset custom delete <name>
+
+**Description**
+
+Delete a custom trivia list.
+
+**Arguments**
+
+* ``<name>``: The name of the custom list to be deleted.
+
+.. _trivia-command-trivia:
+
+^^^^^^
+trivia
+^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia [categories...]
+
+**Description**
+
+Start a trivia session on the specified category.
+
+Multiple categories can be listed, in which case the trivia session 
+will use all of the specified lists to select questions from.
+
+**Arguments**
+
+* ``[categories...]``: The category to play. Can be multiple.
+
+.. _trivia-command-trivia-leaderboard:
+
+^^^^^^^^^^^^^^^^^^
+trivia leaderboard
+^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia leaderboard
+
+**Description**
+
+Shows the trivia leaderboard. Defaults to the top ten in the 
+current server, sorted by total wins. The subcommands provide 
+more customized leaderboards.
+
+.. _trivia-command-trivia-leaderboard-global:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+trivia leaderboard global
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia leaderboard global [sort_by=wins] [top=10]
+
+**Description**
+
+The global trivia leaderboard.
+
+**Arguments**
+
+* ``[sort_by=wins]``: The method by which to sort the leaderboard (defaults to wins). Can be one of: 
+  * ``wins``: total wins
+  * ``avg``: average score
+  * ``total``: total correct answers from all sessions
+  * ``games``: total games played.
+
+* ``[top=10]``: The number of ranks to show on the leaderboard. Defaults to 10
+
+.. _trivia-command-trivia-leaderboard-server:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+trivia leaderboard server
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia leaderboard server [sort_by=wins] [top=10]
+
+**Description**
+
+The trivia leaderboard for this server.
+
+**Arguments**
+
+* ``[sort_by=wins]``: The method by which to sort the leaderboard (defaults to wins). Can be one of: 
+  * ``wins``: total wins
+  * ``avg``: average score
+  * ``total``: total correct answers from all sessions
+  * ``games``: total games played.
+
+* ``[top=10]``: The number of ranks to show on the leaderboard. Defaults to 10
+
+.. _trivia-command-trivia-list:
+
+^^^^^^^^^^^
+trivia list
+^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia list
+
+**Description**
+
+Lists the available trivia categories
+
+.. _trivia-command-trivia-stop:
+
+^^^^^^^^^^^
+trivia stop
+^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia stop
+
+**Description**
+
+Stops an ongoing trivia session.
+
+.. _trivia-list-creation:
+
+----------------------
+Creating your own list
+----------------------
+
+WIP, probably need screenshots here too
```

Comment:
Looks like this part of the guide is not finished yet. Is this intentional?

Reply:
uhh, atm I'm not sure how I want to go about that, especially if we were to consider accepting #4580 as a suggestion to be implemented as that would probably require adding to/rewriting these instructions anyway. I know I also had the thought of having some way of interactively creating a trivia list through a command in Discord and then just responding to prompts but no work has gone into that as of yet. For now, I will probably just remove that section and then work on it down the line